inc/saitunnel.h

```diff
@@ -643,6 +643,15 @@ typedef enum _sai_tunnel_attr_t
      */
     SAI_TUNNEL_ATTR_TERM_TABLE_ENTRY_LIST,
 
+    /**
+     * @brief Packet action when a packet ingress and gets routed back to same tunnel
+     *
+     * @type sai_packet_action_t
+     * @flags CREATE_AND_SET
+     * @default SAI_PACKET_ACTION_FORWARD
```

Comment:
should the default be SAI_PACKET_ACTION_DROP?

Reply:
Most of the usecases I am familiar with are forward for eg MPLS, IPinIP and GRE with L3 payload.